pkg/api/graph/graphview/image_pipeline.go

```diff
@@ -8,6 +8,7 @@ import (
 	osgraph "github.com/openshift/origin/pkg/api/graph"
 	buildedges "github.com/openshift/origin/pkg/build/graph"
 	buildgraph "github.com/openshift/origin/pkg/build/graph/nodes"
+	//imageapi "github.com/openshift/origin/pkg/image/api"
```

Comment:
Deleted.

On Mon, Jul 25, 2016 at 12:35 PM, Ben Parees <notifications@github.com>
wrote:

> In pkg/api/graph/graphview/image_pipeline.go
> <https://github.com/openshift/origin/pull/9411#discussion_r72098002>:
>
> > @@ -8,6 +8,7 @@ import (
> >  	osgraph "github.com/openshift/origin/pkg/api/graph"
> >  	buildedges "github.com/openshift/origin/pkg/build/graph"
> >  	buildgraph "github.com/openshift/origin/pkg/build/graph/nodes"
> > +	//imageapi "github.com/openshift/origin/pkg/image/api"
>
> delete this if we don't need it anymore?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/openshift/origin/pull/9411/files/2eb378ef04518695d44ee2e03bcb9c38a7d94e0c#r72098002>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ADbadPi76lgcWXjK-WBZZ66jizuWsXCwks5qZOW-gaJpZM4I4uXr>
> .
>